applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -639,7 +639,12 @@ static int usb_init(void)
 
 	for (size_t i = 0; i < CONFIG_USB_HID_DEVICE_COUNT; i++) {
 		char name[32];
-		snprintf(name, sizeof(name), CONFIG_USB_HID_DEVICE_NAME "_%d", i);
+		int err = snprintf(name, sizeof(name), CONFIG_USB_HID_DEVICE_NAME "_%d", i);
+
+		if ((err < 0) || (err >= sizeof(name))) {
```

Comment:
We can probably just assert this return value, as the error here means the error in configuration (name too long?).